.prettierignore

```diff
@@ -13,3 +13,6 @@
 coverage
 dist
 node_modules
+
+# drone sign step formats it in a way that prettier doesn't like
+.drone/drone.yml
```

Comment:
Is the signature applied to the whitespace as well?

Another idea is to run `drone sign` w/o the `--save` flag and just manually update the drone config file. It wouldn't update that often to actually matter a copy/paste process after signature generation.

Reply:
I don't think that prettier's opinions on yaml formatting are worth adidng manual steps to it :) 
This config is auto formatted by signing anyhow, maybe it's fine

Reply:
The problem is that some people do have prettier set to auto-format a file when it is opened by the IDE, hence causing corrupted files. But the question still stands... is whitespace considered when doing the signature?

Reply:
yeah, formatting matters, the whole text block is signed.. how bout I add a .prettierignore with drone config at package level?